php/php.api.phpmodule/src/org/netbeans/modules/php/api/validation/ValidationResult.java

```diff
@@ -98,6 +113,20 @@ public boolean hasWarnings() {
 
     }
 
+    /**
+     * Get the first warning.
+     *
+     * @return the first warning or {@code null} if there are no errors
```

Comment:
Should be `[...] no [warnings]` ;)

Reply:
Please, let me know if you are going to change it. If not, I will merge this PR. Thanks.

Reply:
Will fix it, sorry...

Reply:
Fixed.